controllers/shipwrightbuild_controller.go

```diff
@@ -131,13 +149,30 @@ func (r *ShipwrightBuildReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	logger.Info("Applying manifest's resources...")
 	if err := manifest.Apply(); err != nil {
 		logger.Error(err, "Rolling out manifest's resources")
+		apimeta.SetStatusCondition(&b.Status.Conditions, metav1.Condition{
+			Type:    ConditionReady,
+			Status:  metav1.ConditionFalse,
+			Reason:  "Failed",
+			Message: fmt.Sprintf("Reconciling ShipwrightBuild failed: %v", err),
+		})
+		r.Client.Status().Update(ctx, b)
 		return RequeueWithError(err)
 	}
 	if err := r.setFinalizer(ctx, b); err != nil {
 		logger.Info(fmt.Sprintf("%#v", b))
 		logger.Error(err, "Setting the finalizer")
 		return RequeueWithError(err)
 	}
+	apimeta.SetStatusCondition(&b.Status.Conditions, metav1.Condition{
+		Type:    ConditionReady,
+		Status:  metav1.ConditionTrue,
+		Reason:  "Success",
+		Message: "Reconciled ShipwrightBuild successfully",
+	})
+	if err := r.Client.Status().Update(ctx, b); err != nil {
+		logger.Error(err, "Updating ShipwrightBuild status")
```

Comment:
error messages should begin with a lower case letter

Reply:
done